arrow_pd_parser/pa_pd.py

```diff
@@ -114,6 +115,6 @@ def arrow_to_pandas(
     df = arrow_table.to_pandas(
         types_mapper=tm,
         date_as_object=date_as_object,
-        timestamp_as_object=timestamp_as_object,
+        timestamp_as_object=timestamp_as_object,  # Karik this fails
```

Comment:
This doesn't fail for me. Maybe an old version of `pyarrow`? Or a package clash? Tried `poetry lock` -> `poetry install` -> `poetry run pytest tests`